src/main/kotlin/net/irisshaders/lilybot/extensions/util/ThreadControl.kt

```diff
@@ -15,13 +15,14 @@ import com.kotlindiscord.kord.extensions.commands.converters.impl.string
 import com.kotlindiscord.kord.extensions.extensions.Extension
 import com.kotlindiscord.kord.extensions.extensions.publicSlashCommand
 import com.kotlindiscord.kord.extensions.types.edit
-import com.kotlindiscord.kord.extensions.types.respond
 import dev.kord.core.behavior.channel.threads.edit
 import dev.kord.core.entity.channel.thread.ThreadChannel
 import kotlinx.coroutines.flow.toList
-import net.irisshaders.lilybot.utils.DatabaseHelper
+import net.irisshaders.lilybot.utils.getFromConfigPublicResponse
+import net.irisshaders.lilybot.utils.isThread
 import kotlin.time.ExperimentalTime
 
+@Suppress("DuplicatedCode")
```

Comment:
Branch is called deduplication 🤔

Reply:
This is just moving the warning to a more sensible place